HtmlCssSample.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="HtmlCssSample.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Sample Form</title>
+</head>
+
+<body>
+    <nav>
+        <picture class="logo">
+            <img src="./images.png" alt="site logo">
+        </picture>
+        <div>
+            <p>Pricing</p>
+            <p>Solutions</p>
+            <p>Community</p>
+            <p>Resources</p>
+            <p>Contact</p>
+            <section class="buttons">
+                <button class="btn-sign-in">Sign in</button>
+                <button class="btn-register">Register</button>
+            </section>
+
```

Comment:
Use a more semantic structure for the navigation menu, such as a <ul> or <ol> element, instead of a <div> with multiple <p> elements.

Reply:
resolved